src/Sulu/Bundle/SecurityBundle/Domain/Event/UserPasswordResetEvent.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\SecurityBundle\Admin\SecurityAdmin;
+use Sulu\Component\Security\Authentication\UserInterface;
+
+class UserPasswordResetEvent extends DomainEvent
```

Comment:
think this should be `UserPasswordResettedEvent` to be consistent with tho other names